### PR TITLE
firstDraft

### PR DESCRIPTION
**Pull request by ashutoshbondre**
Added 3 files:
1. spec.json (The json data)
2. script.py (The python script which generated the notebook)
3. firstDraft.ipynb (The actual notebook)

### CODE REVIEW

**Review by ellisonbg — CHANGES_REQUESTED** [review]
Talked to Ashutosh today in person. Summary of the discussion:

1. Add `%%html` cells with the converted HTML from the spec file so we can compare our generated HTML.
2. Split the notebook into a series of smaller ones, each with maybe 100 cells (the full one is quite large).
3. Add a section header for each example such as "Example 1" as a Markdown cell.

**Review by mpacer — COMMENTED** [review]
Tried to find all of the little things that will make this more portable and configurable in the long run. Some of my example code would address these issues (at least by making them top level variables instead of hard-coded values).

**Review by ellisonbg — APPROVED** [review]

File: firstDraft.ipynb
Comment (Mhtpyurt) [inline firstDraft.ipynb:1]: H

Comment (Mhtpyurt) [inline firstDraft.ipynb:1]: Aşk

**Review by Mhtpyurt — COMMENTED** [review]
Sevdiğim

**Pull request merged**